pkg/agent/plugin/workloadattestor/docker/cgroup/dockerfinder.go

```diff
@@ -0,0 +1,114 @@
+package cgroup
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// Token is a character or set of characters that are used during parsing of a cgroup entry.
+type Token string
```

Comment:
What's the utility in strongly typing these tokens?

Reply:
I agree there isn't, will remove and unexport the comments.